metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-sqlserver",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
Don't bump versions manually, we have a release_prep workflow that will do this and generate a changelog in a seperate PR :-)